crates/teloxide/src/dispatching/handler_description.rs

```diff
@@ -65,6 +65,8 @@ impl EventKind for Kind {
             EditedMessage,
             ChannelPost,
             EditedChannelPost,
+            MessageReaction,
```

Comment:
> or maybe a test

This can be done with [EnumIter](https://docs.rs/strum/latest/strum/derive.EnumIter.html) derive macro from strum crate (new dep) on `AllowedUpdate` enum or with a hacky test like this:
```rust
let full_set = Kind::full_set();
let allowed_updates_reference = vec![
    Message,
    EditedMessage,
    ChannelPost,
    EditedChannelPost,
    MessageReaction,
    MessageReactionCount,
    InlineQuery,
    ChosenInlineResult,
    CallbackQuery,
    ShippingQuery,
    PreCheckoutQuery,
    Poll,
    PollAnswer,
    MyChatMember,
    ChatMember,
    ChatJoinRequest,
    ChatBoost,
    RemovedChatBoost,
];

for update in allowed_updates_reference {
    // CAUTION: Don't forget to update allowed_updates_reference!
    match update {
        Message | EditedMessage | ChannelPost | EditedChannelPost | MessageReaction
        | MessageReactionCount | InlineQuery | ChosenInlineResult | CallbackQuery
        | ShippingQuery | PreCheckoutQuery | Poll | PollAnswer | MyChatMember
        | ChatMember | ChatJoinRequest | ChatBoost | RemovedChatBoost => {
            assert!(full_set.contains(&Kind(update)))
        }
    }
}
```

Which way is more appropriate?

Reply:
We have chosen the second approach because it avoids exposing extra stuff in the public API.